packages/time-utilities/src/lib/Timestamp.ts

```diff
@@ -158,7 +158,7 @@ export class Timestamp {
 	 * @since 1.0.0
 	 * @param time The time to display in utc
 	 */
-	public displayUTC(time: TimeResolvable): string {
+	public displayUTC(time: TimeResolvable = new Date()): string {
```

Comment:
Redundant as `Timestamp.utc` already sets it to new Date
```suggestion
	public displayUTC(time?: TimeResolvable): string {
```